File: src/main/java/br/com/correios/enderecador/excecao/EnderecadorExcecao.java
package br.com.correios.enderecador.excecao;

public class EnderecadorExcecao extends Exception {
    public EnderecadorExcecao() {
    }

    public EnderecadorExcecao(String mensagem) {
        super(mensagem);
    }

    public EnderecadorExcecao(String msg, Throwable t) {
        super(msg, t);
    }
}

File: src/main/kotlin/br/com/correios/enderecador/excecao/EnderecadorExcecao.kt
package br.com.correios.enderecador.excecao

import java.lang.Exception

open class EnderecadorExcecao : Exception {
    constructor()
    constructor(mensagem: String?) : super(mensagem)
    constructor(msg: String?, t: Throwable?) : super(msg, t)
}
